modules/foreman_operating_system.py

```diff
@@ -191,7 +197,7 @@ def main():
             release_name=dict(),
             description=dict(),
             family=dict(),
```

Comment:
This is also required, at least according to the docs above.

Reply:
Nice spot.